libpostal/mock.js

```diff
@@ -0,0 +1,45 @@
+const _ = require('lodash');
+const fs = require('fs');
+const path = require('path');
+const fixtureFile = path.join(__dirname, './fixture/libpostal_responses.json');
+let fixtures = require(fixtureFile);
+
+/* When `SEED_MOCK_LIBPOSTAL is set, this library will actually
+ * call through to the real libpostal and record the response.
+ * In this way the mock responses can be kept up to date as libpostal changes */
+const SEED_MOCK_LIBPOSTAL = _.has(process, 'env.SEED_MOCK_LIBPOSTAL');
+
+const expand_address = async (address) => {
+
+  // perform some basic normalization on the address string
+  const normalizedAddress = address.trim().toLowerCase();
+
+  // return a mocked response if one is available
+  const mockResponse = _.get(fixtures, normalizedAddress);
+  if (mockResponse) {
+    return Promise.resolve(mockResponse);
+  }
+
+  // if no mock response is available but falling back to libpostal service
+  // is enabled, and return the real response
+  else if (SEED_MOCK_LIBPOSTAL) {
+    const service = require('./service');
```

Comment:
This is where the complexity of supporting multiple ways to use libpostal bites us.

What if there's no libpostal service setting configured? We can't do something like `require('./client')`, because that would create a loop.

This is where I really feel like _only_ supporting libpostal through the libpostal service becomes a huge win. We don't have to worry about handling multiple options.

Reply:
As this is a development only workflow I'm not too concerned?

Reply:
Ok yeah, turns out this flow works fine, I was wrong 😊 

I just ran it after ignoring this branch for 1.5 years and it's really no trouble at all to set up a `pelias.json` with the appropriate `services` entry. In most cases it will already be set up.